packages/babel-parser/src/options.js

```diff
@@ -66,14 +67,20 @@ export const defaultOptions: Options = {
   // When enabled, errors are attached to the AST instead of being directly thrown.
   // Some errors will still throw, because @babel/parser can't always recover.
   errorRecovery: false,
+  inputSourceMap: undefined,
 };
 
 // Interpret and default an options object
 
-export function getOptions(opts: ?Options): Options {
+export function getOptions(opts: ?Options, input: string): Options {
```

Comment:
I there a reason for adding the `input` argument here? It doesn't appear to be used in the function body.